isofit/radiative_transfer/modtran.py

```diff
@@ -329,6 +352,52 @@ def makeSim(self, point, file=None, timeout=None):
                 modtran_config[0]["MODTRANINPUT"]["NAME"] = ""
                 current_config[0]["MODTRANINPUT"]["SPECTRAL"]["FILTNM"] = ""
                 modtran_config[0]["MODTRANINPUT"]["SPECTRAL"]["FILTNM"] = ""
+                if self.multipart_transmittance:
+                    current_config[1]["MODTRANINPUT"]["NAME"] = ""
+                    modtran_config[1]["MODTRANINPUT"]["NAME"] = ""
+                    current_config[1]["MODTRANINPUT"]["SPECTRAL"]["FILTNM"] = ""
+                    modtran_config[1]["MODTRANINPUT"]["SPECTRAL"]["FILTNM"] = ""
+                    current_config[2]["MODTRANINPUT"]["NAME"] = ""
+                    modtran_config[2]["MODTRANINPUT"]["NAME"] = ""
+                    current_config[2]["MODTRANINPUT"]["SPECTRAL"]["FILTNM"] = ""
+                    modtran_config[2]["MODTRANINPUT"]["SPECTRAL"]["FILTNM"] = ""
+                    # Hacky fix to decimel places not matching
+                    modtran_config[0]["MODTRANINPUT"]["AEROSOLS"]["IREGSPC"][0][
+                        "EXTC"
+                    ] = ""
+                    modtran_config[0]["MODTRANINPUT"]["AEROSOLS"]["IREGSPC"][0][
+                        "ABSC"
+                    ] = ""
+                    modtran_config[1]["MODTRANINPUT"]["AEROSOLS"]["IREGSPC"][0][
+                        "EXTC"
+                    ] = ""
+                    modtran_config[1]["MODTRANINPUT"]["AEROSOLS"]["IREGSPC"][0][
+                        "ABSC"
+                    ] = ""
+                    modtran_config[2]["MODTRANINPUT"]["AEROSOLS"]["IREGSPC"][0][
+                        "EXTC"
+                    ] = ""
+                    modtran_config[2]["MODTRANINPUT"]["AEROSOLS"]["IREGSPC"][0][
+                        "ABSC"
+                    ] = ""
+                    current_config[0]["MODTRANINPUT"]["AEROSOLS"]["IREGSPC"][0][
+                        "EXTC"
+                    ] = ""
+                    current_config[0]["MODTRANINPUT"]["AEROSOLS"]["IREGSPC"][0][
+                        "ABSC"
+                    ] = ""
+                    current_config[1]["MODTRANINPUT"]["AEROSOLS"]["IREGSPC"][0][
+                        "EXTC"
+                    ] = ""
+                    current_config[1]["MODTRANINPUT"]["AEROSOLS"]["IREGSPC"][0][
+                        "ABSC"
+                    ] = ""
+                    current_config[2]["MODTRANINPUT"]["AEROSOLS"]["IREGSPC"][0][
+                        "EXTC"
+                    ] = ""
+                    current_config[2]["MODTRANINPUT"]["AEROSOLS"]["IREGSPC"][0][
+                        "ABSC"
+                    ] = ""
                 current_str = json.dumps(current_config)
```

Comment:
@jammont - can you help clean this up so it's not so redundant and easier to update?

Reply:
Posted: https://github.com/reginaeckert/isofit/pull/1

Reply:
Looks good! I accepted the pull request.